### PR TITLE
[176936463] returns 401 for incorrect login credentials

### PR DESCRIPTION
**Pull request by cliberman**

### CODE REVIEW

**Review by thecomputerguyjeff — APPROVED** [review]

File: client/src/components/LoginPage.js [hunk 1]
```diff
@@ -41,12 +41,15 @@ export default class Login extends Component {
...
+            ).then(res => {
+                if (res.status === 200)
+                    return res.json();
+                throw 'not 200'
+            })
                 .then((response) => {
                     this.setState({
                         user: response
                     });
-                    this.props.setUser(response)
```
Comment (DevorahLevi) [inline client/src/components/LoginPage.js]: Don't we still need this line to set the user after they are logged in?

Comment (thecomputerguyjeff) [reply client/src/components/LoginPage.js]: Agreed

**Review by thecomputerguyjeff — CHANGES_REQUESTED** [review]

**Comment by cliberman** [comment]
I put that line back and recommitted but I'm not sure it's showing up here?

**Comment by DevorahLevi** [comment]
> I put that line back and recommitted but I'm not sure it's showing up here?

I think that since you put it back to how it was before, it is not different from the master branch and therefore not showing up in the files changed.

**Review by DevorahLevi — APPROVED** [review]

**Pull request merged**